source/Playlist.cpp

```diff
@@ -55,9 +55,17 @@ void Playlist::Load(const DataNode &node)
 		else if(key == "location")
 			location.Load(child);
 		else if(key == "priority" && hasValue)
+		{
 			priority = max<unsigned>(0, child.Value(1));
+			if(child.Value(1) < 0)
+				child.PrintTrace("Warning: Negative weight sepcified for priority.");
```

Comment:
```suggestion
				child.PrintTrace("Warning: Ignoring negative priority:");
```
~~Just support negative priorities~~